pages/boardgames/detail/[id].tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Header from "components/Header";
+import Footer from "components/Common/Footer";
+import BoardDetailsPage from "components/BoardDetailsPage/BoardDetailsPage";
+import Seo from "components/Common/Seo";
+import database from "middlewares/dbForFrontend";
+import { getBoardgame } from "db/bg";
+import { getBoardGameDetails } from "db/boardGameDetails";
+
+interface Props {
+  boardData: {
+    descriptionUrl: string;
+    imageUrl: string;
+    nameValue: string;
+    yearValue: string;
+  };
+}
+
+const BoardgameDetailPage = ({ boardData }: Props) => {
+  return (
+    <div>
+      <Seo
+        isHomepage={false}
+        title="Comprehensive Ain'tboard board game collection"
+        description="Explore our comprehensive board game collection, featuring top picks from BoardGameGeek and Ain'tboard rankings. Dive into a world of strategic fun, with games rated and reviewed by the board gaming community. Whether you're a casual player or a hardcore enthusiast, discover a variety of games to suit all preferences. Our curated selection is updated regularly to include the latest high-ranking entries. Unleash the gamer in you and find the perfect addition to your collection today!"
+        noIndex="noindex"
+        noFollow="nofollow"
+      />
+      <Header headerStyle={{ minHeight: "10rem" }} />
+      <BoardDetailsPage boardData={boardData} />
+      <Footer />
+    </div>
+  );
+};
+
+export default BoardgameDetailPage;
+export async function getServerSideProps(context: any) {
+  try {
+    const db = await database();
+    const { id } = context.params;
+    const boardgameResponse = await getBoardgame(db, id);
+    // console.log(boardgameResponse)
+    const [descUrl, imgUrl, yearValue, nameValue] = await getBoardGameDetails(
+      boardgameResponse.response.data.ID
+    );
+    console.log(imgUrl, yearValue);
+    const boardData = {
+      descriptionUrl: descUrl,
+      imageUrl: imgUrl,
+      yearValue: yearValue,
+      nameValue: nameValue,
```

Comment:
looks good but why rename the object props? 
```suggestion
    const boardData = {
descUrl,
imgUrl,
yearValue,
nameValue,
```
we can just do this no?

Reply:
This was breaking before - but I will test again

Reply:
i guess bcos the one that consumes it doesnt match the data structure

Reply:
Yes - I will leave it as is because when I change it the client side breaks